src/test/java/Home20work.java

```diff
@@ -1,30 +1,38 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.ElementClickInterceptedException;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
 import org.testng.annotations.Test;
 
 import java.time.Duration;
 
 
 public class Home20work {
+    static WebDriverWait wait=null;
+
     @Test
-    public void clearCurrentQueue() {
-        starup();
-        //login();
-    }
+    private static void registration1() {
 
-    private void starup() {
         WebDriver driver = new ChromeDriver();
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+        //driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10))
+        wait = new WebDriverWait(driver, Duration.ofSeconds(10));
         String url = "https://bbb.testpro.io/";
         driver.get(url);
-        WebElement emailField = driver.findElement(By.cssSelector("[type='email']"));
-        emailField.clear();
-        emailField.sendKeys("karkisuman0330@gmail.com");
-        WebElement passwordField = driver.findElement(By.cssSelector("[type='password']"));
-        passwordField.clear();
-        passwordField.sendKeys("te$t$tudent");
+
+        WebElement registrationButton1 = wait.until(ExpectedConditions.elementToBeClickable(By.id("[id='hel']")));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/213858949-db5ba7e1-16e2-493e-be3b-e9a0db205a97.png)

Hint 

     `WebElement registrationButton1 = wait.until(ExpectedConditions.elementToBeClickable(By.id("hel")));`